crates/jstzd/src/task/octez_node.rs

```diff
@@ -0,0 +1,264 @@
+use super::Task;
+use anyhow::Result;
+use async_dropper::AsyncDrop;
+use async_trait::async_trait;
+use std::{fs::File, path::PathBuf, sync::Arc};
+use tokio::{runtime::Handle, sync::RwLock};
+
+#[cfg(test)]
+use tests::{Child, InnerOctezNode};
+
+#[cfg(not(test))]
+use octez::OctezNode as InnerOctezNode;
+#[cfg(not(test))]
+use std::process::Child;
+
+pub struct OctezNodeConfig {
+    binary_path: PathBuf,
+    data_dir: PathBuf,
+    network: String,
+    http_endpoint: String,
```

Comment:
In #565, `http_endpoint` is being used as the p2p endpoint but we shouldn't need to set that. This field can be removed.

Reply:
Removed